idaes/surrogate/tests/test_pysmo_surrogate.py

```diff
@@ -0,0 +1,2110 @@
+         [0.690326213554, 0.569364994374, 0.982068847698],
+         [-0.961163301329, 0.499471920546, 0.936855365038]]),
+        columns=["x1", "x2", "z1"])
+
+    @pytest.fixture(scope="class")
+    def pysmo_trainer_krg(self):
+        # Test end-to-end workflow with a simple problem.
+        bnds = {"x1": (-1.5, 1.5), "x2": (-1.5, 1.5)}
+        pysmo_trainer_krg = PysmoKrigingTrainer(
+            input_labels=["x1", "x2"],
+            output_labels=["z1"],
+            input_bounds=bnds,
+            training_dataframe=TestKrigingWorkflow.training_data)
+        pysmo_trainer_krg.config.regularization = False
+        pysmo_trainer_krg.config.numerical_gradients = False
+
+        np.random.seed(0)
+        res = pysmo_trainer_krg.train_surrogate()
+
+        return res
+
+    def test_pysmo_trainer_krg_results(self, pysmo_trainer_krg):
+        assert pysmo_trainer_krg._data is not None
+        assert pysmo_trainer_krg.model_type == "kriging"
+        assert pysmo_trainer_krg.num_outputs == 1
+        assert pysmo_trainer_krg.output_labels == ['z1']
+        assert pysmo_trainer_krg.input_labels == ['x1', 'x2']
+        assert len(pysmo_trainer_krg._data)== 1
+
+    def test_pysmo_trainer_krg_object(self, pysmo_trainer_krg):
+        bnds = {"x1": (-1.5, 1.5), "x2": (-1.5, 1.5)}
+        input_labels=["x1", "x2"]
+        output_labels=["z1"]
+        input_bounds=bnds
+        pysmo_object = PysmoSurrogate(pysmo_trainer_krg, input_labels, output_labels, bnds)
+        assert isinstance(pysmo_object, PysmoSurrogate)
+        assert pysmo_object._input_labels == ["x1", "x2"]
+        assert pysmo_object._output_labels == ["z1"]
+        assert pysmo_object._input_bounds == {
+            "x1": (-1.5, 1.5), "x2": (-1.5, 1.5)}
+
+        # Check populating a block to finish workflow
+        blk = SurrogateBlock(concrete=True)
+
+        blk.build_model(pysmo_object)
+
+        assert isinstance(blk.inputs, Var)
+        assert blk.inputs["x1"].bounds == (-1.5, 1.5)
+        assert blk.inputs["x2"].bounds == (-1.5, 1.5)
+        assert isinstance(blk.outputs, Var)
+        assert blk.outputs["z1"].bounds == (None, None)
+        assert isinstance(blk.pysmo_constraint, Constraint)
+        assert len(blk.pysmo_constraint) == 1
+
+    def test_metrics(self, pysmo_trainer_krg):
+        bnds = {"x1": (-1.5, 1.5), "x2": (-1.5, 1.5)}
+        input_labels=["x1", "x2"]
+        output_labels=["z1"]
+        input_bounds=bnds
+        pysmo_object = PysmoSurrogate(pysmo_trainer_krg, input_labels, output_labels, bnds)
+
+        metrics = compute_fit_metrics(pysmo_object, TestRBFWorkflow.training_data)
+
+        assert isinstance(metrics, dict)
+
+        assert metrics["z1"]["R2"] == pytest.approx(
+            pysmo_trainer_krg._data['z1'].metrics['R2'], rel=1e-8)
+        assert metrics["z1"]["RMSE"] == pytest.approx(
+            pysmo_trainer_krg._data['z1'].metrics['RMSE'], rel=1e-8)
+
+
```

Comment:
Extra blank lines. I would suggest you run `black` on the code in this PR, as we are in the process of doing that anyway,

Reply:
Done